updater/library/src/config.rs

```diff
@@ -0,0 +1,68 @@
+// This file handles the global config for the updater library.
+
+use std::cell::RefCell;
+
+use crate::updater::AppConfig;
+
+const DEFAULT_BASE_URL: &'static str = "https://shorebird-code-push-api-cypqazu4da-uc.a.run.app";
+const DEFAULT_CHANNEL: &'static str = "stable";
+
+thread_local!(static CONFIG: RefCell<Option<ResolvedConfig>> = RefCell::new(None));
+
+pub fn with_config<F, R>(f: F) -> R
+where
+    F: FnOnce(&ResolvedConfig) -> R,
+{
+    CONFIG
+        .try_with(|config| {
+            let config = config.borrow();
+            let config = config
+                .as_ref()
+                .expect("Must call updater_init before using the updater library.");
```

Comment:
expect is like "unwrap" but with a message.  Unwrap takes the insides out of an Option<T> or Result<T> and hands you back the T.